Google Sheet Missing Outputs bug (#27358)
demisto/content: Packs/GoogleSheets/ReleaseNotes/1_0_26.md

#### Integrations

##### Google Sheets

> - Fixed an issue where response was substituted to output only if *echo_spreadsheet* argument was set to true.

Reviewer: ```suggestion
- Fixed an issue where outputs were only written to context when *echo_spreadsheet* argument was set to true.
```
